lib/turn.rb

```diff
@@ -0,0 +1,26 @@
+class Turn
+  attr_reader :guess,
+              :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == card.answer
+      true
+    else
+      false
+    end
+  end
+
+  def feedback
+    if correct? == true
```

Comment:
Great job using the `correct?` method here, but remember since the `correct?` method returns a boolean, we do not need to compare it to a boolean in an `if` statement. This method could have just been written like so:

```ruby
def feedback
  if correct?
    "Correct!"
  else
    "Incorrect."
  end
end
```